konlpy/data.py

```diff
@@ -111,7 +111,7 @@ def load(resource_url, format='auto'):
     return resource_val
 
 
-class PathPointer(object):
+class PathPointer:
```

Comment:
This would be fine if we're going to fully drop Python 2.x support beginning with this commit, but if it is not the case I'm afraid if this change (not inheriting from `object`) could introduce some unwanted break.

Reply:
@nyanye, could you exclude this change from the pull request? Then I think everything else is fine to be merged without further discussion.

Reply:
I agree, I'll revert this one :)

Reply:
ah, I was too naive. all 4 classes that explicitly inherited 'object' are now fixed.